### ISSUE TITLE
Update to 3.61.1

### ISSUE THREAD

**Issue by PanderMusubi** [flathub/no.mifi.losslesscut#62]
Please upgrade to version 3.61.1, see also https://github.com/mifi/lossless-cut/releases/tag/v3.61.1

See also https://github.com/mifi/lossless-cut/issues/1477 and https://github.com/flathub/no.mifi.losslesscut/pulls

For users of Pop!_OS, flathub is the preferred way they can install losslesscut.

**Comment by muelli** [comment]
yeah, somebody needs to fix up the builds https://github.com/flathub/no.mifi.losslesscut/pull/59#issuecomment-2039398659
Let me know if you need any help!

**Comment by Lissard** [comment]
Hi,

Is there any update on this?  The current version is now [3.62.0](https://github.com/mifi/lossless-cut/releases/tag/v3.62.0) compared to the flatpak (3.55.1) which is very outdated now.

Thanks

**Comment by rammjet** [comment]
3.63.0 now.  Would be nice.

**Comment by tjanez** [comment]
I've tried to update to version 3.64.0 in https://github.com/flathub/no.mifi.losslesscut/pull/73.

The issue I've encountered are countless `RequestError: getaddrinfo EAI_AGAIN registry.yarnpkg.com` errors at `yarn install` step.

I would need help from the author of the [Yarn 2 support in `flatpak-node-generator` PR](https://github.com/flatpak/flatpak-builder-tools/pull/252#issuecomment-2467875948) to advise on which command to use to make the Yarn package fetching and installation work with the latest Yarn version (LosslessCut currently uses version 4.4.0).
